src/moscot/backends/ott/solver.py

```diff
@@ -114,6 +155,11 @@ def is_low_rank(self) -> bool:
         """Whether the :attr:`solver` is low-rank."""
         return self.rank > -1
 
+    @property
+    def graph_in_linear_term(self) -> bool:
```

Comment:
Where is this used? Imho the solver doesn't need to care about this and can be removed.